Resources/Prototypes/DeltaV/Roles/Jobs/Command/admin_assistant.yml

```diff
@@ -0,0 +1,75 @@
+- type: job
+  id: AdministrativeAssistant
+  name: job-name-admin-assistant
+  description: job-description-admin-assistant
+  playTimeTracker: JobAdminAssistant
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Logistics
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
+      department: Engineering
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
+      department: Medical
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
+      department: Security
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
+      department: Epistemics 
+      time: 18000 # 5 hours
+    - !type:DepartmentTimeRequirement
+      department: Command
+      time: 108000 # 30 hours
+    - !type:OverallPlaytimeRequirement 
+      time: 108000 # 30 hours
+    - !type:WhitelistRequirement
+  weight: 20
+  startingGear: AdminAssistantGear
+  icon: "JobIconAdminAssitant"
+  requireAdminNotify: true
+  joinNotifyCrew: true
+  supervisors: job-supervisors-command
+  canBeAntag: false
+  access:
```

Comment:
This is gross. This is Captain lite, some of these like theatre rooms are *not* relevant to their job.